core/tests/webdriverio_utils/general.js

```diff
@@ -305,6 +305,17 @@ var goOffline = async function() {
   });
 };
 
+var getFullStackTraceForDeferredCall = async function(func, errStack) {
```

Comment:
Sorry, let's rename this. The current semantics are to just get the stack trace but we should make it clear that it's actually calling the function.

How about callFunctionAndCollectFullStackTraceOnError()?

Reply:
done.